creator-node/src/utils/cluster/clusterUtilsForPrimary.ts

```diff
@@ -1,9 +1,24 @@
 import type { Cluster } from 'cluster'
 import type Logger from 'bunyan'
+import type { MetricToRecord } from '../../services/prometheusMonitoring/types'
 
 import { isClusterEnabled } from './clusterUtils'
 const cluster: Cluster = require('cluster')
 
+// Lazy-load prometheus utils to avoid DiskManager tests throwing errors when loading it too soon doens't allow everything to be mocked properly
```

Comment:
seems weird haha - am tempted to just let this one go, do you think there's any other way around this?

Reply:
unfortunately I spent too much time on this for this PR and a past issue 😅  some dependency that DiskManager tests try to mock just don't allow it to import certain things. I think I did make a change to DiskManager at one point that fixed this, but it was a very involved change that was also kinda ugly. if it helps, prom-client lazy loads a module in this same way to avoid a different but similar-ish problem ([code](https://github.com/siimon/prom-client/blob/master/lib/cluster.js#L14-L20))

Reply:
hahaha its not ideal but if it works locally and in prod then w/e